services/download/download.sh

```diff
@@ -4,9 +4,14 @@ set -Eeuo pipefail
 
 mkdir -p /data/.cache /data/StableDiffusion /data/Codeformer /data/GFPGAN /data/ESRGAN /data/BSRGAN /data/RealESRGAN /data/SwinIR /data/LDSR /data/ScuNET /data/embeddings
 
+ARIA2C_OPTS=""
+if [ "`ip -6 addr`" == "" ]; then
+    ARIA2C_OPTS="${ARIA2C_OPTS} --disable-ipv6"
+fi
+
```

Comment:
is this really necessary? just disable ipv6 totally, also, is it possible to have this flag only on the v1.5 model in the `links.txt`? maybe `disable-ipv6=true`?

Reply:
Checking IPV6 makes me sleep better :-)
In contrast to https://github.com/moby/moby/issues/32850 which seems to wait over 5 years, and nobody knows if tomorrow there will be another user presenting some edge case of docker use on theirs system ipv6 exclusive.
OK. I've explained myself. Now... Your solution with flag seems to be better. I am sorry - I do not have even basic knowledge about aria2c. I will look onto it in manual and probably will commit change without unnecessary code.

Best regards

Reply:
ayy man no worries man, I don't know `aria2c` either :')

I just googled `cli download utility linux` and it was third after `curl` and `wget`, both of them failed on my extremely slow and unstable internet connection.

Reply:
> is this really necessary? just disable ipv6 totally, also, is it possible to have this flag only on the v1.5 model in the `links.txt`? maybe `disable-ipv6=true`?

There is no ipv6 specific option per link ( https://aria2.github.io/manual/en/html/aria2c.html#input-file )
After some research yesterday I think `--disable-ipv6` option for entire command is only way to go.
I've commited changes so merge is now possible.